src/demo-app/stepper/stepper-demo.html

```diff
@@ -0,0 +1,27 @@
+<h2>Horizontal Stepper Demo</h2>
+<mat-horizontal-stepper>
+  <mat-step *ngFor="let step of steps" [label]="step.label">
+    <md-input-container>
+      <input mdInput placeholder="Answer" [(ngModel)]="step.content">
+    </md-input-container>
+  </mat-step>
+</mat-horizontal-stepper>
```

Comment:
Just noticed that the stepper selectors need to be added to the compatibility mode list as well
(which enforces that you use either `md-` or `mat-` and not both)